app/components/ShopCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { TShop } from '@/types/shop-types'
+
+interface IProps {
+  shop: TShop
+  handleCardClick: (shop: TShop) => any
+  handleKeyPress: (event: React.KeyboardEvent<HTMLLIElement>, shop: TShop) => any
+}
+
+export default function ShopCard(props: IProps) {
+  return (
+    <li
```

Comment:
This probably should probably be part of ShopList since the card won't always be used in an list